NodeJS-HW4/src/db/models/userGroupModel.ts

```diff
@@ -0,0 +1,16 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '..';
+
+const UserGroup = sequelize.define('usergroups', {
```

Comment:
You should not do this manually. Plz, use `belongsToMany`/`hasMany` associations:

https://sequelize.org/docs/v6/advanced-association-concepts/advanced-many-to-many/